<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_type_de_moyen</class>
 <widget class="QDialog" name="Dialog_type_de_moyen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edition des types de moyens</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../openjardin.qrc">:/images/motoculteur.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_idType_de_moyen">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>TYPES DE MOYENS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_Type_de_moyen"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_Designation"/>
       </item>
       <item row="2" column="2">
        <widget class="QTextEdit" name="textEdit_Commentaires"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Commentaires</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>DÃ©signation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nouveau">
         <property name="text">
          <string>Nouveau type de moyen</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/add_48.png</normaloff>:/images/add_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Supprimer">
         <property name="text">
          <string>Supprimer</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/remove_48.png</normaloff>:/images/remove_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Valider">
         <property name="text">
          <string>Valider</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/Valider.png</normaloff>:/images/Valider.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Annuler">
         <property name="text">
          <string>Annuler</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/icones/quitter.png</normaloff>:/images/icones/quitter.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../openjardin.qrc"/>
 </resources>
 <connections/>
</ui>
